openid-4-verifiable-presentations-1_0.md

```diff
@@ -331,6 +331,24 @@ The following is a non-normative example of a transaction data content, after ba
 }
 ```
 
+`attachments`:
```

Comment:
Please add to IANA considerations too, [Appendix D.2](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#appendix-D.2)

I think you also want to add it to the list of allowed parameters for Browser DC API?

Reply:
good point, added to both